Microsoft.Research/Contracts/System.Core.Contracts/System.Linq.Expressions.SwitchCase.cs

```diff
@@ -44,8 +45,11 @@ sealed public partial class SwitchCase
     #region Methods and constructors
     private SwitchCase() {}
 
+    [Pure]
     public SwitchCase Update (IEnumerable<Expression> testValues, Expression body)
     {
+      Contract.Requires(body != null);
```

Comment:
:bulb: You can definitely require `testValues` not be null here (enforced by the implementation).